README.md

```diff
@@ -14,3 +14,22 @@ With docker-compose
 ```
 SUBSTRAT_CONFIG_FILE=values-docker-compose.yaml make test
 ```
+
+# Tests Configuration
+
+The tests suite requires a Substra network up and running. The network can be started
+either with skaffold, manually or through docker-compose.
```

Comment:
Maybe these two lines could be moved to a "Prerequisites" section?

Reply:
Yes, done.